spec/dummy/db/schema.rb

```diff
@@ -227,6 +243,13 @@
 
   add_index "spree_log_entries", ["source_id", "source_type"], name: "index_spree_log_entries_on_source_id_and_source_type"
 
+  create_table "spree_mail_methods", force: true do |t|
+    t.string   "environment"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.